src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy

```diff
@@ -122,12 +126,21 @@ class NUnit extends ConventionTask {
         project.file(getReportFolder())
     }
 
+    File getLogFolderImpl() {
+        if (logFolder)
+            project.file(logFolder)
+        else
+            return null
+    }
+
     @OutputFile
     File getTestReportPath() {
         // for the non-default nunit tasks, ensure we write the report in a separate file
-        // TODO: Do we need to supply prefix when user has specified its own report file name?
-        def reportFileNamePrefix = name == 'nunit' ? '' : name
```

Comment:
> When making this change I was taking into account the fact that if user didn't specify report file, he doesn't really need it

I hear you, but we can't be sure we don't break lots of usages out there, that rely on the xml being named by the plugin automatically, based on the task name. That's why I'd say we should aim at keeping a backwards compat behavior if possible and if it doesn't prevent us from evolving. 

@gluck @bcorne Feedback on this?